src/bin/vip-wp.js

```diff
@@ -131,6 +176,8 @@ commandWrapper( {
 		const { id: appId, name: appName } = opts.app;
 		const { id: envId, type: envName } = opts.env;
 
+		let cmdGuid;
```

Comment:
I think we can use `cliCommand.guid` directly 🤔

Reply:
cliCommand is defined inside the `subShellRl.on( 'line', async line => {` event handler, but isn't available in the `subShellRl.on( 'SIGINT', () => {` handler,  hence setting a variable outside of that.